pom.xml

```diff
@@ -48,14 +48,14 @@
         <version.com.networknt>1.0.86</version.com.networknt>
         <version.org.slf4j>1.7.25</version.org.slf4j>
         <version.com.fasterxml.jackson>2.15.2</version.com.fasterxml.jackson>
-        <version.javax.validation>2.0.1.Final</version.javax.validation>
+        <version.javax.validation>3.0.2</version.javax.validation>
```

Comment:
nit : should this maven property name changed to reflect jakarta - version.jakarta.validation?

Reply:
updated